src/Integrations/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -288,6 +311,125 @@ function ($exceptionHandler, $scope, $args) use ($rootSpan, $integration) {
             }
         );
 
+        // Used by Laravel >= 5.0
+        \DDTrace\hook_method(
+            'Illuminate\Auth\SessionGuard',
+            'attempt',
+            null,
+            function ($This, $scope, $args, $loginSuccess) use ($rootSpan, $integration) {
+                if ($loginSuccess) {
+                    return;
+                }
+
+                if (!function_exists('\datadog\appsec\track_user_login_failure_event'))
+                {
+                    return;
+                }
+                \datadog\appsec\track_user_login_failure_event(null, false, [], true);
```

Comment:
Is it not possible to provide the user id in the event of login failure? Or does the RFC suggest otherwise?

Reply:
Milestone 0 does not require id on this event. I have an idea of how to get it for failure logins but I will do that on another PR.